app/services/induction/change_mentor.rb

```diff
@@ -49,4 +45,24 @@ def send_training_materials_if_needed
       send_training_materials
     end
   end
+
+  def send_training_materials
+    MentorMailer.with(
+      mentor_profile:,
+      ect_name: induction_record.participant_profile.user.full_name,
+      cip_materials_name: cip_materials.name,
+      sit_name:,
+    ).training_materials
+     .deliver_later
+  end
+
+  def sit_mentor_out_of_frozen_cohort?
```

Comment:
The method name doesn't really reflect the implementation as I understand it:

We return `false` if the ECT is in a cohort with payments frozen, otherwise apply the normal "eligible to change" rules?

I think this method is to do with eligibility to amend the cohort, and the payments frozen logic is just one of the conditions.

```suggestion
  def mentor_can_change_cohort?
```

Reply:
Not sure I follow.
A mentor like this one would be in a frozen cohort, with billable declarations, not completed declarations... and all the rest of conditions to be called unfinished participant.
This method is to check if we must move them out of that frozen cohort after being assigned an ECT.

Reply:
Happy to rename it to anything else though.

Reply:
I think there's ambiguity in the prefix `sit_` does this mean _to place_ or a type of Induction Tutor?

The implementation is concerned with the rules of whether a mentor can or should change cohort, only one of the conditionals is concerned with _payments frozen_, there's a further check on `eligible_to_change_cohort_and_continue_training?` which has nothing to do with frozen cohorts, so the more general name I suggested was just an idea of how to make the code more readable and perhaps a more generalised wrapper name around the two internal conditionals.

Reply:
Yes, we usually use the verb sit meaning placing the participant in a cohort.

The first conditional is about the ECT (not the mentor) being in a payments_frozen cohort. The method name is about the mentor (not the ECT).

In any case, shall we use  `change_mentor_cohort?` instead?
If so, feel free to commit my 2 suggestions.

Reply:
@ltello I think we should avoid `sit_` as a prefix as it could easily be mistaken for induction tutor in this codebase.

I understand what the method does, my suggestion was that we mask the various conditionals, some of which have little to do with frozen payments from the method name. I'll commit my original suggestion.